C-Plus-Plus/dp/unbounded_knapsack.cpp

```diff
@@ -0,0 +1,58 @@
+/**
+ * PROBLEM STATEMENT 
+ * Finding the maximum value of the items that can be popacked into the knapsack if it is possible to take one item multiple times
+ * DYNAMIC PROGRAMMING
+ * Time Complexity O(n^2)
```

Comment:
If you are referring an online CP problem, add the link

Reply:
I am not referring to any online CP problem

Reply:
I wrote it as the heading as there are three types of knapsack problem